html-aam/dir-role.tentative.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+  <title>HTML-AAM dir Element Role Verification Test</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+
+<p>Tentative test for the expected role mappings of the obsolete <code>dir</code> element.
+  The computedrole mappings are defined in <a href="https://w3c.github.io/html-aam/">HTML-AAM</a>.<p>
+
+<p>Merge the outcome of this test into roles.html in the appropriate alphabetical order, when it is no longer considered tentative:</p>
+
+<dir data-testname="el-dir" data-expectedrole="list" class="ex"><li>x</li><li>x</li></dir>
```

Comment:
Since it's obsolete/deprecated, and all the implementations map it to generic, why not just leave it generic in roles.html? I don't see a lot of value in suggesting the implementations map a _never-implemented_ and _no-longer-recommended_ element.

Reply:
Sorry I didn't notice this was obsolete previously.

Reply:
Curious if @aleventhal or @jcsteh have an opinion on the role mapping addition. Either way, it's probably fine to check in as tentative…

Reply:
> Since it's obsolete/deprecated, and all the implementations map it to generic, why not just leave it generic in roles.html? I don't see a lot of value in suggesting the implementations map a _never-implemented_ and _no-longer-recommended_ element.

@cookiecrook last year webkit updated `dir` to follow the HTML spec for how it is supposed to render, this is where the topic of whether it should also have proper aam mappings was brought up - https://github.com/whatwg/html/issues/9296

this is closing the loop that webkit started ;)

Reply:
Fair enough. Thanks for the additional background.